test/npu-xrt/ctrl_packet_reconfig/run.lit

```diff
@@ -8,3 +8,4 @@
 // RUN: clang %S/test.cpp -o test.exe -std=c++11 -Wall %xrt_flags -lrt -lstdc++ -lboost_program_options -lboost_filesystem
 // RUN: %run_on_npu ./test.exe -x aie.xclbin -k MLIR_AIE -i insts.txt | FileCheck %s
 // CHECK: PASS!
+// XFAIL: *
```

Comment:
I guess this is because you are stacking PRs? (my suggestion...)

Reply:
This test was the control packet reconfiguration prototype which loads control packets from a txt file that I generated offline. The goal is to have compiler passes generate the txt, but the passes are in this PR https://github.com/Xilinx/mlir-aie/pull/1753. Once we land both PRs then I will land a separate PR to rewrite this test.